server/controller/module/sendMail.ts

```diff
@@ -0,0 +1,59 @@
+import { createTransport } from "nodemailer";
+import * as dotenv from "dotenv";
+import path from "path";
+
+//The below config setup was unable to be resolved. Need to connect and see on this
+
+const config = dotenv.config({
+  path: path.resolve(__dirname, "./env"),
+}).parsed;
+
+export default async function sendMail(reci: String[]) {
+  var transport = {
+    host: config?.SMTP_HOST,
+    secureConnection: false,
+    port: config?.SMTP_PORT,
+    tls: {
+      ciphers: "SSLv3",
+    },
+    auth: {
+      user: config?.MAILER_USER,
+      pass: config?.MAILER_PASS,
+    },
+  };
+  var transporter = createTransport(transport);
+
+  var toObj = "";
+
+  //The below loop is implementation logic. It has to be refactored
+  for (var i = 0; i < reci.length; i++) {
+    if (i == reci.length - 1) {
+      toObj += reci[i];
+      continue;
+    }
+    toObj += reci[i] + ", ";
+  }
```

Comment:
I do not think we need this loop instead think of the scenario where we simply pass `list<toEmailIds>` as parameters and it simply sends a batch email via smtp server